test/jdk/javax/swing/JColorChooser/Test4759306.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2002, 2008, 2024 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2002, 2008, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This line is wrong, we only use the first and last years. Please delete "2008, "